README.md

```diff
@@ -2217,15 +2217,21 @@ Other Style Guides
 ## Accessors
 
   - [23.1](#23.1) <a name='23.1'></a> Accessor functions for properties are not required.
-  - [23.2](#23.2) <a name='23.2'></a> If you do make accessor functions use getVal() and setVal('hello').
+  - [23.2](#23.2) <a name='23.2'></a> Do not use JavaScript getters/setters as they cause unexpected side-effects and are harder to test, mantain and reason about. Instead, if you do make accessor functions, use getVal() and setVal('hello').
```

Comment:
`side effects` is two words, `s/mantain/maintain`, and please add the Oxford comma after "maintain".